rfc/composite_table_availability.pseudocode

```diff
@@ -0,0 +1,116 @@
+
+PhysicalTableSchema:
+	getFieldName(Column c)
+
+DataSource:
```

Comment:
The class [DataSource](https://github.com/yahoo/fili/blob/master/fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/datasource/DataSource.java) already exists to hold `PhysicalTables` in `DruidQuery`, we might have to find a different name?